app/services/bulk_tagging/document_type_tagger.rb

```diff
@@ -0,0 +1,27 @@
+module BulkTagging
+  class DocumentTypeTagger
+    def self.call(taxon_base_path:, document_type:)
+      new(taxon_base_path: taxon_base_path, document_type: document_type).call
+    end
+
+    def initialize(taxon_base_path:, document_type:)
+      @taxon_base_path = taxon_base_path
+      @document_type = document_type
+    end
+
+    def call
```

Comment:
Looks like you could save some lines by just changing the above line to `self.call(taxon_base_path:, document_type:)` and removing the `self.call` and `initialize` thing, which might be clearer.

Reply:
I agree that this initialize->call thing is a bit weird, but it is used all over the place for services in content tagger so I used it for consistency sake.